Modules/Microsoft365DSC/DSCResources/MSFT_IntuneSettingCatalogCustomPolicyWindows10/MSFT_IntuneSettingCatalogCustomPolicyWindows10.psm1

```diff
@@ -107,24 +107,30 @@ function Get-TargetResource
         {
             Write-Verbose -Message "Could not find an Intune Setting Catalog Custom Policy for Windows10 with Id {$Id}"
 
-            if (-Not [string]::IsNullOrEmpty($Name))
+            if (-not [string]::IsNullOrEmpty($Name))
             {
                 $getValue = Get-MgBetaDeviceManagementConfigurationPolicy `
-                    -Filter "Name eq '$Name' and Platforms eq 'windows10'" `
+                    -Filter "Name eq '$Name'" `
                     -ErrorAction SilentlyContinue | Where-Object `
-                    -FilterScript {[String]::IsNullOrWhiteSpace($_.TemplateReference.TemplateId)}
-                if ($getValue.count -gt 1)
+                    -FilterScript {
+                        $_.Platforms -in @('windows10', 'macOS', 'iOS') -and
+                        $_.Technologies -in @('mdm', 'windows10XManagement', 'appleRemoteManagement') -and
```

Comment:
This resource is supposed to be used only for Windows, why are you adding more platforms?

Reply:
That's called late-night sickness... Ahh. Will update it in an instant.

Reply:
@ricmestre Terribly sorry about the confusion, now back to the previous version with only exporting Windows policies.